apps/server/src/schemas/apiKey.schema.ts

```diff
@@ -0,0 +1,24 @@
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+import { HydratedDocument } from 'mongoose';
+
+export type ApiKeyDocument = HydratedDocument<ApiKey>;
+
+@Schema()
+export class ApiKey {
+  @Prop({ required: true })
+  name: string;
+
+  @Prop({ enum: ['full_access', 'sending_access'], default: 'sending_access' })
+  permission: string;
+
+  @Prop({ type: Date, default: Date.now })
+  createdAt: Date;
+
+  @Prop()
+  domainId?: string;
+
+  @Prop({ required: true })
+  token: string;
```

Comment:
Could you update please removing this `token` field, because we don't want to store the `token` value of an API key, just return it once in creation for the user to store and use.